guide/case-studies/bitcoin-core-app.md

```diff
@@ -0,0 +1,245 @@
+
+<div class="image-slide-gallery">
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/bitcoin-core-app/v-0-1.png"
+   retina = "assets/images/guide/case-studies/bitcoin-core-app/v-0-1@2x.png"
+   alt-text = "Screenshot of the original bitcoin wallet software"
+   caption = "Version 0.1 from 2009."
+   width = 600
+   height = 282
+   layout = "shadow"
+%}
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/bitcoin-core-app/v-0-5-2.png"
+   retina = "assets/images/guide/case-studies/bitcoin-core-app/v-0-5-2@2x.png"
+   alt-text = "Screenshot of the 2011 rewrite"
+   caption = "Version 0.5.2 from 2011."
+   width = 600
+   height = 368
+   layout = "shadow"
+%}
+
+{% include picture.html
+   image = "assets/images/guide/case-studies/bitcoin-core-app/v-26.png"
+   retina = "assets/images/guide/case-studies/bitcoin-core-app/v-26@2x.png"
+   alt-text = "Screenshot of the 2023 version of the software"
+   caption = "Version 26 from 2023."
+   width = 600
+   height = 384
+   layout = "shadow"
+%}
+
+</div>
+
+
+The Bitcoin Core project plays an essential role in the ecosystem. It not only serves as the reference implementation of the Bitcoin protocol, but it is also the only implementation on the node side. The software includes node and wallet features and is broadly used by miners, businesses and enthusiasts, as well as a more casual audience.
+
+## Goals
+
+There are four main goals of this project:
+
+- Empower people to run a node easily on a mobile device.
+- Expand wallet features wallet with the latest best practices and developments.
+- Provide more useful tools for experts and developers.
+- Build a solid design foundation and public blueprint.
+
+### Nodes on mobile devices
+
+<div class="center">
+
+{% include image.html
+   image = "assets/images/guide/case-studies/bitcoin-core-app/node-on-device.jpg"
+   retina = "assets/images/guide/case-studies/bitcoin-core-app/node-on-device@2x.jpg"
+   alt-text = "Android smartphone running the software, showing the block clock"
+   layout = "float-right-desktop"
+   width = 400
+   height = 416
+%}
+
+A major goal is to make running a node easier and more rewarding, since that is an essential function in the bitcoin network. Running a node means actively helping to verify consensus rules, for example that coins are not spent which do not exist. Node runners also get to personally manage, verify and broadcast transactions, which is a great privacy benefit.
```

Comment:
```suggestion
A major goal is to make running a node easier and more rewarding, since that is an essential function in the bitcoin network. Running a node means actively helping to verify consensus rules, for example coins are not spent which do not exist. Node runners also get to personally manage, verify and broadcast transactions, which is a great privacy benefit.
```